lib/galaxy/tool_util/loader_directory.py

```diff
@@ -21,7 +21,7 @@
 TOOL_REGEX = re.compile(r"<tool\s")
 DATA_MANAGER_REGEX = re.compile(r"\stool_type=\"manage_data\"")
 
-YAML_EXTENSIONS = [".yaml", ".yml", ".json"]
+YAML_EXTENSIONS = [".yaml", ".yml"]
 CWL_EXTENSIONS = YAML_EXTENSIONS + [".cwl"]
```

Comment:
Does adding .json to CWL_EXTENSIONS in the next line to make up for this help anything or does that revert the performance. I would be fine with that change but I'm not sure it helps. I've seen CWL json files I think.

Reply:
This does not have any effect on the performance since: looks_like_a_tool also rund `looks_like_a_tool_cwl` (because it is  in `BETA_TOOL_CHECKERS`) which considers the `CWL_EXTENSIONS`: https://github.com/galaxyproject/galaxy/blob/cdf9253908c7b7e82e654186e0fabef728719d67/lib/galaxy/tool_util/loader_directory.py#L150